tests/st/bgp/test_ipip.py

```diff
@@ -287,3 +287,51 @@ def get_tunl_tx(self, host):
         match = re.search(r'RX packets:(\d+) ',
                           output)
         return int(match.group(1))
+
+    def test_ipip_gce(self):
+        """
+        Test IPIP routing on simulated GCE instances.
+        """
+        with DockerHost('host1',
+                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
+                        simulate_gce_routing=True,
+                        start_calico=False) as host1, \
+             DockerHost('host2',
+                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
+                        simulate_gce_routing=True,
+                        start_calico=False) as host2:
+
+            host1.start_calico_node()
+            host2.start_calico_node()
+
+            # We are simulating GCE instance routing, where there is a router
+            # between the instances, and each instance has a /32 address that
+            # appears not to be directly connected to any subnet.  Hence we
+            # also need to enable IP-in-IP to get from one host to the other.
+            host1.enable_ipip()
+            host2.enable_ipip()
```

Comment:
IP pools are globally configured and not host specific, so I think you can drop one these as the second call won't actually make any more updates.